src/components/Employee/Compensation/useCompensation.ts

```diff
@@ -8,78 +8,104 @@ import { yearlyRate } from '@/helpers/payRateCalculator'
 export const CompensationSchema = z
   .object({
     jobTitle: z.string().min(1),
-    // adjustForMinimumWage can be true or false
     adjustForMinimumWage: z.boolean(),
-    // If adjustForMinimumWage is true, minimumWageId is required
```

Comment:
thank you for removing this, the comment doesn't match the schema now or else this would have to be a variant